entrypoint.sh

```diff
@@ -24,7 +33,8 @@ if [[ "$1" == "bump:"* ]]
         | grep new_version | sed -r s,"^.*=",,`
 
     echo "Bumping to version $new_version"
-    git push
+    git tag -a $new_version -m "Release $new_version"
```

Comment:
If we do that, we add the tag before the PR is actually merged to master. If I remember well the tag will be lost when merging.

Reply:
Ah yes right, I remove it. 
My other idea would be to create a workflow to add the tag and create the draft release whenever a branch that begins by "release/" is merged. What do you think ?

Reply:
that could be great! the draft release link could be sent in the #python-sdk channel.